rust/crd/src/constants.rs

```diff
@@ -1,5 +1,18 @@
 //! This module provides constants that are shared via multiple crates.
 
+pub const FIELD_MANAGER_SCOPE: &str = "sparkcluster";
+
+pub const PORT_NAME_WEB: &str = "web";
+pub const PORT_NAME_MASTER: &str = "master";
```

Comment:
I think we have the `web` ports now mostly called `http`. 
I would rename the master port to something like `protocol` or `spark` and reuse it for worker.

Reply:
Done.

Reply:
:+1: on `spark` (or `spark-master` if we need to differentiate the master port specifically)